source/_components/google_assistant.markdown

```diff
@@ -137,31 +137,31 @@ If you're not using Linux, you can use sites such as [this one](https://www.brow
 
 ### {% linkable_title Setup %}
 
-1. Create a new project in the [developer console](https://console.actions.google.com/).
-  a. Add/Import project
-  b. Click on `BUILD` on the `Smart home` card
+1. Create a new project in the [developer console](https://console.actions.google.com/).  
+  a. Add/Import project  
+  b. Click on `BUILD` on the `Smart home` card  
   c. Type in your home assistant url: `https://[YOUR HOME ASSISTANT URL:PORT]/api/google_assistant`, replace the `[YOUR HOME ASSISTANT URL:PORT]` with the domain / ip address and the port under which your Home Assistant is reachable.
```

Comment:
This should read:
Click `Build your Action`, click `Add Action(s)`, click `Add your first action`, select your language

Type in your home assistant url: `https://[YOUR HOME ASSISTANT URL:PORT]/api/google_assistant`, replace the `[YOUR HOME ASSISTANT URL:PORT]` with the domain / ip address and the port under which your Home Assistant is reachable. If you have set `api_password:` add this password to the URL `https://[YOUR HOME ASSISTANT URL:PORT]/api/google_assistant?api_password=[YOUR API PASSWORD]`)

Reply:
This adds more explicit instruction for adding the action, as well as mentioning that you need the `api_password` on this step too if you have one.